Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11.8-alpine3.18
+LABEL maintainer="ivchenkomykhailo@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py", "0.0.0.0:8000"]
```

Comment:
Why you need `0.0.0.0:8000`?

Reply:
Because I haven't understood that I don't need it if I do not run a server))
Fixed